app/locale/en.coffee

```diff
@@ -1434,36 +1434,46 @@
     earliest_incomplete: "Earliest incomplete level"
     latest_complete: "Latest completed level"
     enroll_student: "Enroll student"
+    apply_license: "Apply License"
     course_progress: "Course Progress"
     not_applicable: "N/A"
     edit: "edit"
     edit_2: "Edit"
     remove: "remove"
-    latest_completed: "Latest completed"
+    latest_completed: "Latest completed:" # {change}
     sort_by: "Sort by"
     progress: "Progress"
     completed: "Completed"
     started: "Started"
     click_to_view_progress: "click to view progress"
     no_progress: "No progress"
     select_course: "Select course to view"
+    students_not_assigned: "Students who have not been assigned {{courseName}}"
     course_overview: "Course Overview"
     copy_class_code: "Copy Class Code"
     class_code_blurb: "Students can join your class using this Class Code. No email address is required when creating a Student account with this Class Code."
     copy_class_url: "Copy Class URL"
     class_join_url_blurb: "You can also post this unique class URL to a shared webpage."
     add_students_manually: "Invite Students by Email"
     bulk_assign: "Bulk-assign"
+    assigned_msg_1: "{{numberAssigned}} students were assigned {{courseName}}."
+    assigned_msg_2: "{{numberEnrolled}} licenses were applied."
+    assigned_msg_3: "You have {{remainingSpots}} licenses remaining."
+    assign_course: "Assign Course"
+    not_assigned_modal_title: "Courses were not assigned"
+    not_assigned_modal_body_1: "You do not have enough licenses to assign additional Courses to all {{selected}} selected students."
+    not_assigned_modal_body_2: "You only have {{totalSpotsAvailable}} licenses ({{unenrolledStudents}} students did not have an active license)."
```

Comment:
I'd change this to "available licenses" instead of "licenses", for the case where they have licenses that unlock in the future.